src/udev/udev-builtin-hwdb.c

```diff
@@ -99,6 +99,18 @@ static int udev_builtin_hwdb_search(sd_device *dev, sd_device *srcdev,
                         last = true;
                 }
 
+                if (streq(dsubsys, "firewire")) {
+                        const char *devname;
+
+                        // The node device has character device while the unit device doesn't.
+                        if (!sd_device_get_devname(d, &devname) && strlen(devname) > 0) {
+                                // The parent of node is 1394 OHCI controller in PCI bus. Let us
+                                // cancel walkthrough.
```

Comment:
Please use `/* */` style for the comments instead of `//`.

Reply:
I overlook it. I'm okay to fix it in revised patchset.